zebedee-cms/src/main/java/com/github/onsdigital/zebedee/api/Root.java

```diff
@@ -122,9 +125,16 @@ public static void init() {
 
         SlackNotification.alarm("Zebedee has just started. Ensure an administrator has logged in.");
 
-        loadExistingCollectionsIntoScheduler();
+
+        try {
+            Collections.CollectionList collections = zebedee.getCollections().list();
+            alertOnInProgressCollections(collections, new SlackNotifier());
+            loadExistingCollectionsIntoScheduler(collections);
+        } catch (IOException e) {
+            error().logException(e, "zebedee root: failed to load collections list check in progress approvals");
```

Comment:
My only thought is if the loading fails for some other reason other than ‘in progress’ ones, would the error message be misleading?

Reply:
Good spot Jess - I have updated the log message so that it's not specific to any one function call that uses the collection list.